src/Documentation/ComponentDocs/CardDocs/CardDocs.css

```diff
@@ -0,0 +1,60 @@
+.card {
+  border: 1px solid grey;
+  border-radius: 5px;
+}
+.card-I {
```

Comment:
I have a suggestion. I think card-I and card-II are not clear, can you change them so that they describe the type of card or css properties

Reply:
Yes, I have to work a lot more on cards. Will do that👍